sync.js

```diff
@@ -244,6 +258,41 @@ class Sync extends events.EventEmitter {
     return peer.sync
   }
 
+  connectWebsocket (url, projectKey) {
+    let {
+      protocol,
+      hostname: host,
+      port: rawPort
+    } = new URL(url)
+
+    let port = rawPort
+    if (!port) {
+      port = (protocol === 'ws:') ? 80 : 443
```

Comment:
I think maybe we should enforce SSL? I imagine it could be easy for someone setting up a project to accidentally write `ws://` instead of `wss://` and we would not want to open up a way to accidentally expose unencrypted data. What is the use-case for an unencrypted websocket connection?

Reply:
I imagine an unencrypted connection could be useful as an option for testing and for people hosing mapeo web servers on community networks.

Maybe we can emit a warning when generating the settings file?

Reply:
Right now the hypercore data gets encrypted anyways, and the plan for the next iteration of sync is to wrap it all in NOISE, so it might not matter if it's `ws` vs `wss`.